tck/event-sourced-entity.js

```diff
@@ -65,7 +65,10 @@ function process(request, state, context) {
     effects.forEach(effect =>
       reply.addEffect(two.service.methods.Call, { id: effect.id }, effect.synchronous)
     )
-  return reply
+
+  return new Promise((resolve, reject) => {
```

Comment:
this test that the mechanism is working as expected (note that you can increase the timeout and the tests are still passing)